tests/analytics/test_anonymizer.py

```diff
@@ -1,10 +1,8 @@
 import pytest
 
-from great_expectations.analytics.anonymizer import Anonymizer
+from great_expectations.analytics.anonymizer import anonymize
 
 
 @pytest.mark.unit
 def test_anonymizer_anonymize():
-    anonymizer = Anonymizer(salt="salt")
-    expected_hash = "162902aed084586c5ae5be2973bcbe35"
-    assert anonymizer.anonymize("string") == expected_hash
+    assert anonymize("string") == "b45cffe084dd3d20d928bee85e7b0f21"
```

Comment:
Simple test but just going to keep it around in case anyone accidentally breaks `anonymize` down the line?